src/sourcemap-codec.ts

```diff
@@ -84,12 +88,12 @@ export function encode ( decoded ) {
 		return line.map( encodeSegment ).join( ',' );
 	}).join( ';' );
 
-	function encodeSegment ( segment ) {
+	function encodeSegment ( segment: SourceMapSegment ): string {
 		if ( !segment.length ) {
-			return segment;
+			return '';
```

Comment:
Previously this returned an empty array, which would then get `toString`-ed to an empty string by `join`. TypeScript didn't like that, so I just return an empty string now. 😃